classes/PrestaShopCollection.php

```diff
@@ -336,7 +336,7 @@ public function getAll($display_query = false)
         $this->is_hydrated = true;
 
         $alias = $this->generateAlias();
-        //$this->query->select($alias.'.*');
+        $this->query->select('*, ' . $alias . '.`' . $this->definition['primary'] . '`');
```

Comment:
```suggestion
        $this->query->select($alias . '*');
```

Is enough, since `$this->definition['primary']` is already in `*` :thinking: